assets/dev/scss/frontend/_grid.scss

```diff
@@ -137,6 +137,11 @@
 	> .elementor-element {
 		width: 100%;
 	}
+
+	// Fix: When adding margin to column of carousel widgets, it breaks the UI.
```

Comment:
Why the: "// Fix:"?
+
I would elaborate on the problem a bit, it's not understood what exactly breaks.